commands/roles.go

```diff
@@ -106,6 +108,8 @@ func rolesMemberCmdF(c client.Client, _ *cobra.Command, args []string) error {
 
 			printer.Print(fmt.Sprintf("System admin role revoked for user %q", args[i]))
 		}
+
+		printer.Print(fmt.Sprintf("Updated %s roles: %s", user.Email, user.Roles))
```

Comment:
Here we should do something similar, merging the print with the one inside the if condition and printing the roles in `newRoles` instead of the `user` ones.